src/cache.jl

```diff
@@ -94,7 +94,11 @@ end
 updatecache(absentmodule::Symbol, server) = updatecache([absentmodule], server)
 
 function updatecache(absentmodules::Vector{Symbol}, server)
+    server.cp_lock && return
+
+    server.cp_lock = true
     o,i, p = readandwrite(`$JULIA_HOME/julia -e "include(\"packages/LanguageServer/src/cache.jl\");
+    delete!(Base.ENV, \"JULIA_PKGDIR\");
```

Comment:
@davidanthoff, I seem to need this to be able to search my user package dir, is this specific to me/linux?